src/lib/components/ILSMenu/ILSMenu.js

```diff
@@ -115,6 +116,13 @@ class ILSMenu extends Component {
                   </div>
                 </Menu.Item>
                 <Menu.Menu position="right">
+                  <Menu.Item
+                    onClick={() =>
+                      goTo(FrontSiteRoutes.documentsListWithQuery(''))
+                    }
+                  >
+                    Go to the catalogue
```

Comment:
What about simply `Catalogue`? (to reduce the complexity of the menu)

Reply:
In cds-ils the logo says `CERN Library Catalogue` and I thought having a second button that just says `Catalogue` would be confusing.

Reply:
What about `Search`? I would be in favor of one word too... maybe quick chat with the team? during stand up?